go/ct/spc/specification.go

```diff
@@ -2024,11 +2026,15 @@ func getRulesForCall(op OpCode, revision Revision, warm, zeroValue bool, opEffec
 }
 
 func callEffect(s *st.State, addrAccessCost vm.Gas, op OpCode) {
+	isDelegateCall := op == DELEGATECALL
+
 	gas := s.Stack.Pop()
 	target := s.Stack.Pop()
 	var value U256
 	if op == CALL {
 		value = s.Stack.Pop()
+	} else if isDelegateCall {
+		value = s.Accounts.GetBalance(s.CallContext.AccountAddress)
```

Comment:
This specification is wrong: it states that with a delegate call all the money of the calling account is to be transferred.

However, what should be passed as a value is the CallValue of the current call (`s.CallContext.Value`).